src/parseTools.js

```diff
@@ -10,6 +10,8 @@
 
 globalThis.FOUR_GB = 4 * 1024 * 1024 * 1024;
 globalThis.WASM_PAGE_SIZE = 64 * 1024;
+// Represents a browser version that is not supported at all.
+globalThis.TARGET_NOT_SUPPORTED = 0x7FFFFFFF;
```

Comment:
Is the name supposed to be `UNSUPPORTED` as you have it below?

Reply:
In JavaScript this constant is called `TARGET_NOT_SUPPORTED`.

In python its called `feature_matrix.UNSUPPORTED`.   This change just makes it consistent since python was using both previously.